matcher:
  ros__parameters:
    wms:
      url: 'http://localhost:8080/wms'
      version: '1.1.1'
      layer: 'WorldImagery'
      srs: 'EPSG:4326'  # util.get_bbox currently only supports EPSG:4326 --> don't change this setting
    misc:
      affine: True  # Consider setting to True if camera is known to be nadir-facing
      gimbal_projection: True
      max_map_radius: 1000
      map_radius_meters_default: 400
      update_map_center_threshold: 20
      update_map_radius_threshold: 50
  superglue:  # Recommended outdoor config from SuperGlue repo's README.md
    superpoint:
      nms_radius: 3
      keypoint_threshold: 0.005
      max_keypoints: 2048
    superglue:
      weights: 'outdoor'
      sinkhorn_iterations: 20
      match_threshold: 0.2